src/turnkeyml/build/export.py

```diff
@@ -403,6 +364,82 @@ def fire(self, state: fs.State, opset: int = build.DEFAULT_ONNX_OPSET):
         return state
 
 
+class VerifyOnnxExporter(stage.Stage):
+    """
+    Stage that runs a parity test on an input PyTorch model and an ONNX
+    file derived from that model.
+
+    Note that the derived ONNX file is discarded by the verification API,
+    so we can't use it in downstream Stages. To use this stage inline with
```

Comment:
I'm curious: What is the overhead of this "export" that doesn't generate any onnx files?
Feels odd that we have to "export" twice.

Reply:
Yeah that's just how the validator works :( 

Thus why it is a separate stage now: you only go through the hassle of the extra export if you really want the data.